FogCreek/Glitch-Community
# enchanted-contraption: add collections propTypes and defaultProps to CreateCollectionPop

## Links
* [enchanted-contraption.glitch.me](https://enchanted-contraption.glitch.me)
* [Sentry event](https://sentry.io/organizations/fog-creek-software/issues/1043808827/?project=1246508&query=is%3Aunresolved)
* [Manuscript: cannot read property filter of null](https://glitch.fogbugz.com/f/cases/3328655/Create-collection-pop-cannot-read-property-filter-of-undefined)

## Changes:
`maybeCollections` is `null` by default, this defines `propTypes` and `defaultProps` for `collections`, so we always expect an array.

## How To Test:
Open the add project to collection pop, click "add to a new collection". You shouldn't see any errors.
